api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java

```diff
@@ -22,29 +25,40 @@
  * @since 1.5
  * @see OpenmrsMetadata
  */
+@MappedSuperclass
 public abstract class BaseOpenmrsMetadata extends BaseOpenmrsObject implements OpenmrsMetadata {
 	
 	//***** Properties *****
+	@Column(name = "name", nullable = false, length = 255)
 	@Field
 	private String name;
 	
+	@Column(name = "description", length = 255)
```

Comment:
255 is the most common length value for description property in hibernate mapping files for classes derived from BaseOpenmrsMetadata. In some of them however, larger values are used, like: 500, 1024, 65535 or 2147483847. Should it be changed for one of these values?